### ISSUE TITLE
[Dark theme updates] Font style updates

### ISSUE THREAD

**Issue by setch-l** [planetary-social/nos#939]
Update the font sizes for the following font styles: 

Headline 
Clarity City - Semi-bold 
Size = 17pt 
Leading = 22pt 

Subhead
SF Pro Text - Regular 
Size = 15pt
Leading = 20pt 

Footnote 
SF Pro Text - Regular 
Size = 13pt
Leading = 18pt

Body 
SF Pro Text Regular 
Size = 17pt
Leading = 22pt 

Design Reference files to verify both the color and size are correct: 

https://www.figma.com/file/s0qf4VmyQygydP8MIQazZc/Nos?type=design&node-id=5228%3A31642&mode=design&t=K6Jm3QTxDsITuN7p-1

https://www.figma.com/file/s0qf4VmyQygydP8MIQazZc/Nos?type=design&node-id=5228%3A31758&mode=design&t=K6Jm3QTxDsITuN7p-1

https://www.figma.com/file/s0qf4VmyQygydP8MIQazZc/Nos?type=design&node-id=5228%3A31858&mode=design&t=K6Jm3QTxDsITuN7p-1

**Comment by Chardot** [comment]
> Had some questions that I asked @Chardot on Slack; he made updates to some of the Figma screens and labels which makes the description here outdated. I'll update it to remove the copied styles so we have one source of truth (Figma).

Yes, all Subhead text appears in Medium weight on the designs but were labeled as Regular. I fixed the labeling in the callouts:

![Screenshot 2024-03-12 at 18 52 47](https://github.com/planetary-social/nos/assets/477835/98c51a8a-654a-476c-9c96-ef0ca7b0bbb7)

**Comment by setch-l** [comment]
@joshuatbrown - I forgot part of the story in this ticket. it has now been updated with the font styles.
